src/status_im/notifications/core.cljs

```diff
@@ -147,23 +159,26 @@
                 (when event
                   (handle-notification-event event))))))
 
-  (defn on-notification-opened []
+  (defn setup-on-notification-opened-callback []
     (.. firebase
         notifications
         (onNotificationOpened handle-notification-event)))
 
   (defn init []
-    (on-refresh-fcm-token)
-    (on-notification)
-    (on-notification-opened)
+    (setup-token-refresh-callback)
+    (setup-on-message-callback)
+    (setup-on-notification-opened-callback)
     (when platform/android?
       (create-notification-channel))))
 
 (fx/defn process-stored-event [cofx address]
   (when-not platform/desktop?
-    (let [to (get-in cofx [:db :accounts/accounts address :public-key])
-          from (get-in cofx [:db :push-notifications/stored to])]
-      (when from
+    (let [current-account (get-in cofx [:db :account/account])
```

Comment:
@yenda The problem here is that when `process-stored-event` is called, were already signed in, so `:accounts/accounts` isn't available anymore (I wasn't aware of that before looking into this), so we need to rely on `:account/account` instead.